### PR TITLE
Added release notes for Nov and Dec 2021.

### PR DESCRIPTION
**Pull request by bengo1990**
Added release notes for Nov and Dec 2021.

https://docs.vualto.com/projects/vudrm/en/feature-vudrm-1846-release-notes-update/

### CODE REVIEW

**Review by askolodny — DISMISSED** [review]

**Review by somerobmitchell — DISMISSED** [review]

File: docs/source/ReleaseNotes.md [hunk 1]
```diff
@@ -16,6 +18,26 @@
...
+
+* Internal improvements to our Stats API.
+
+* Internal improvements to our CPIX API.
+
+* Internal improvements to our KP API.
+
 ## October 2021
 
 * Improvements to our FairPlay, Widevine and KP API systems to be compatible with Token v2.
```
Comment (PerritonBranch) [inline docs/source/ReleaseNotes.md:43]:
```suggestion
* Improvements to our FairPlay, Widevine, and KP API systems to be compatible with Token v2.
```

**Review by PerritonBranch — DISMISSED** [review]

File: docs/source/ReleaseNotes.md [hunk 2]
```diff
@@ -16,9 +18,29 @@
...
 
+## December 2021
+
+* Internal improvements to our Config API. Added functionality to integrate with JW Player systems.
+
+* Internal improvements to our CPIX API.
+
+## November 2021
+
+* Improvements to VUDRM Admin. Video.js and Dash.js added as an option for the public player.
```
Comment (PerritonBranch) [inline docs/source/ReleaseNotes.md:29]: Should this be Studio DRM admin now?

**Review by PerritonBranch — CHANGES_REQUESTED** [review]

File: docs/source/ReleaseNotes.md (hunk 2 above)
Comment (bengo1990) [reply docs/source/ReleaseNotes.md:29]: Probably not - the site, URLs and such haven't been rebranded, so could add confusion?

Happy to do it if you like though.

Comment (PerritonBranch) [reply docs/source/ReleaseNotes.md:29]: @askolodny Any opinions on this? Do we want to leave it as "VUDRM Admin" for now and then when all of these docs are actually moved over change it to "Studio DRM" then?

**Review by somerobmitchell — APPROVED** [review]

**Pull request merged**